src/app/store/datatypes.ts

```diff
@@ -87,7 +87,7 @@ export class UserState {
   id: number | null;
   whiteList: WhiteList[];
   blackList: BlackList[];
-  settings: Settings;
+  settings: Settings = new Settings();
```

Comment:
remove it from here and update the `initialState` in store. `users.reducers.ts`.
That is where all the state is being initialized.

Reply:
Okay,